standard/json.lua

```diff
@@ -23,7 +23,8 @@ end
 ---@param pretty boolean?
 ---@return string
 function Json.stringify(obj, pretty)
```

Comment:
Do we ever use the `pretty` option?

Reply:
on the git no, outside would need checking